docs/guides/place-my-order.md

```diff
@@ -382,6 +382,13 @@ We have now created a model and fixtures (for testing without an API) with a fol
 |   |   ├── test.js
 ```
 
+We also need to specify that the restaurant list can be filtered to restaurants in a queried city and state by updating `src/models/restaurants.js`:
+
+@sourceref ../../guides/place-my-order/steps/create-test/restaurants_model.js
+@highlight 15-16
+
+Above we use 'set.props.dotNotation' since our queries for these nested properties will be in the MongoDB-style 'dot notation' format required by the backend.
```

Comment:
Think I should add a link to the Mongo docs that describe why dot notation is relevant?